src/procedures/proc_functions.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright FalkorDB Ltd. 2023 - present
+ * Licensed under the Server Side Public License v1 (SSPLv1).
+ */
+
+
+#pragma once
+
+#include "proc_ctx.h"
+
+// Returns all the system functions. 
+// This procedure yields the function name, description and signature.
```

Comment:
please add an example of a single returned function

Reply:
Done